### ISSUE TITLE
Accessibility Bug: When the theme is switched to ‘High contrast Black’ the UI elements in the properties section is not visible.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3116]
Fundamental Core component: "Dialog"

Description: When the theme is switched to ‘High contrast Black’ the UI elements in the properties section is not visible.

Expected: All the UI elements should get adapt to the High contrast Black theme when the theme is changed.

Attached the screenshot:
In Fiori 3 theme:

![image](https://user-images.githubusercontent.com/32538291/91553284-c29ad480-e94a-11ea-9288-cc21694a1e1d.png)

In High Contrast Black theme:

![image](https://user-images.githubusercontent.com/32538291/91553349-dfcfa300-e94a-11ea-85a0-b1a9e62fb2d0.png)

**Comment by droshev** [comment]
this was a documentation issue which was already addressed
![Screen Shot 2020-09-28 at 8 47 30 PM](https://user-images.githubusercontent.com/4380815/94499927-ec564e00-01cb-11eb-9fe0-d65636fa677b.png)
